harness/determined/deploy/aws/deployment_types/vpc.py

```diff
@@ -38,3 +38,14 @@ class FSx(VPCBase):
 class EFS(VPCBase):
     template = "efs.yaml"
     deployment_type = constants.deployment_types.EFS
+
+
+class Lore(VPCBase):
+    template = "lore.yaml"
+    deployment_type = constants.deployment_types.LORE
+
+    def before_deploy_print(self) -> None:
+        super().before_deploy_print()
+        lore_tag = self.parameters[constants.cloudformation.LORE_VERSION] or "latest"
```

Comment:
maybe a dumb question: what's going to be a Lore "version"? will it be the same as Determined version? can this just be the determined version?

Reply:
it's a short version of the commit hash that lore was built on. we don't have a set versioning system yet.
we also have a "latest" tag
it does not sync or match with det versions